README.md

```diff
@@ -21,5 +21,46 @@ client build <directory> --output my-workspace
 ### Template content in a directory and push to a registry location
 `client build <directory> --push --destination localhost:5000/myartifacts:latest`
 
+## Getting Started
+
+1. Create a new directory. 
+2. Add the content to be uploaded in the directory (can be files of any content types).
+3. Create a json doc where the value of each kv pair is the path to each file within the directory. Multiple json docs can be used to create deep graphs, but a graph must only have one root. Multiple json docs in a build directory is for advanced use cases. Most use cases do not need more than one json doc. 
+
+Example json doc:
+```
+{
+    "fish": "fish.jpg",
+    "text": "subdir1/file.txt",
+    "fish2": "subdir1/fish2.jpg"
+}
+```
+4. Create a dataset-config.yaml outside of the content directory that references the relative paths from within the content directory to each file. Add user defined key value pairs as subkeys to the `section`. Each file should have as many attributes as possible. Groupings can be created by using the `*` wildcard. 
+
+Example dataset-config.yaml:
+```
+kind: DataSetConfiguration
+apiVersion: client.uor-framework.io/v1alpha1
+files:
+  file: fish.jpg
+  attributes:
+    animal: fish
+    habitat: ocean
+    size: small
+    color: blue
+  file: level1/file.txt
+  attributes:
+    fiction: true  
+    genre: science fiction
+```
+5. Run the UOR client build command referencing the dataset config, the content directory, and optionally push to a registry location.
+
+```
+client build --dsconfig dataset-config.yaml content-dir --push --destination localhost:5000/test/dataset:latest
+```
+
+6. Optionally inspect the OCI manifest of the dataset:
+
+curl <servername>:<port>/v2/<namespace>/<repo>/manifests/<digest or tag>
```

Comment:
Let's continue with the example:

```shell
curl -H "Accept: application/vnd.oci.image.manifest.v1+json" http://localhost:5000/v2/test/dataset/manifests/latest 
```

Reply:
Ah yeah, I forget this because go-containerregistry/registry doesnt require the content type declaration in the header (i use gcr in testing)